dlp/snippets/inspect_content.py

```diff
@@ -310,6 +310,103 @@ def inspect_table(
 
 # [END dlp_inspect_table]
 
+
+# [START dlp_inspect_column_values_w_custom_hotwords]
+from typing import Dict, List, Union  # noqa: E402, I100
+
+import google.cloud.dlp  # noqa: F811, E402
+
+
+def inspect_column_values_w_custom_hotwords(
+    project: str,
+    table_header: List[str],
+    table_rows: List[List[str]],
+    info_types: List[str],
+    custom_hotword: str,
+) -> None:
+    """Uses the Data Loss Prevention API to inspect table data using built-in
+    infoType detectors, excluding columns that match a custom hot-word.
```

Comment:
please fix typos throughout the file: change `hot-word` to `hotword`

Reply:
Updated!